mlx/traceability.py

```diff
@@ -9,7 +9,7 @@
 
 from __future__ import print_function
 import re
-from sphinx.util.compat import Directive
+from docutils.parsers.rst import Directive
```

Comment:
How about you wrap this into at least version check of sphinxs to keep backwards compatibility

Reply:
CI passes on older versions of sphinx as well (see CI), so i think this `sphinx.util.compat` was a courtesy package proxying to `docutils.parsers.rst` which will soon be removed


Note: `docutils.parsers.rst.Directive` was added in docutils in 2006 (https://sourceforge.net/p/docutils/code/4667/)

Reply:
Oky. :+1:  to merge then